packages/ui-dom-utils/src/getClassList.ts

```diff
@@ -23,10 +23,16 @@
  */
 
 import { findDOMNode } from './findDOMNode'
+import React from 'react'
 
-let classListShimmed = false
+type ClassListApiType = {
```

Comment:
For me the name `type ClassListApiType` is a bit redundant, I'd drop the Type from it and change it to simply
be `ClassListApi`.